packages/filesystem/src/browser/filesystem-watcher.ts

```diff
@@ -167,16 +167,16 @@ export class FileSystemWatcher implements Disposable {
             });
     }
 
-    protected createWatchOptions(): Promise<WatchOptions> {
-        return this.getIgnored().then(ignored => ({
+    protected createWatchOptions(uri?: string): Promise<WatchOptions> {
+        return this.getIgnored(uri).then(ignored => ({
```

Comment:
Why do we have the "URI" as optional when the parent calling this method is not "Optional"